### PR TITLE
Install instructions

### LINKED ISSUE #95: Install instructions
Write complete installation instructions for Windows server 2012 R2 :cry: :sob: :crying_cat_face:

### PR DESCRIPTION
**Pull request by SpaceLenore**
Adds complete install instructions both as markdown and pdf. Closes #95

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+För att starta servern ska du ha öppnat cmd i serverkatalogen och kör kommandot `node index.js`. Detta startar servern i denna terminalen. För att avsluta / stoppa servern trycker du på tangentbordet `Ctrl + c`. Notera att den inte kommer starta automatiskt om ni stoppar den eller startar om värdmaskinen. 
+
+### Tillåt servern i brandväggen
+
+När man startar servern för första gången så brukar Windows be en ge systemet tillgång till internet, godkänn detta. I övrigt kan du behöva justera dina brandväggar och öppna portar beroende på hur ditt nätverk ser ut. 
+
+## Underhåll och uppdateringar
+
+Hela projektet finns på GitHub med historik av alla förändringar. Det är rekommenderat att bihehålla git verisionshantering för projektet ifall det i framtiden kommer att vidareutvecklas.
```
Comment (Glassinatorn) [inline INSTALL.md]: "bibehålla" not "bihehålla"

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+Stå i mappen `Backend` och kör kommandot `npm install`
+
+## Kör konfigurationsskripten
+
+Flytta till mappen `setup`, kör skriptet `setup.sh` och följ instruktionerna på skärmen.   
+När det är klart så kör du `node createAdminAccount.js` för att skapa den första administratörsanvändaren.
+
+## Underhåll och uppdateringar
+
+Hela projektet finns på GitHub med historik av alla förändringar. Det är rekommenderat att bihehålla git verisionshantering för projektet ifall det i framtiden kommer att vidareutvecklas.
```
Comment (Glassinatorn) [inline INSTALL.md]: same spelling misstake as above

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+# Installationsguide för Baga Aqua System API
+
+### 2019-05-21
+
+# Förord
+
+Denna Installationsguide är för Baga Aqua Systems API. Systemet är byggt i NodeJS med MongoDB som databas. Arbetet är utfört av studenter på Blekinge Tekniska Högskola som en del i kursen PA1416 under vårenterminen 2019 åt Baga Water Technology AB. Denna guide är till för att underlätta vid installation av systemet. Läsaren förväntas ha kunskaper om den egna miljön som installationen sker i.
```
Comment (Glassinatorn) [inline INSTALL.md]: As the rest of the installation guide uses language that is directed directly towards the reader, it is recommended to change "läsaren" to "du"

Comment (SpaceLenore) [reply INSTALL.md]: fixed in commit 7127c28

File: INSTALL.md (hunk 2 above)
Comment (SpaceLenore) [reply INSTALL.md]: fixed in commit 7127c28

File: INSTALL.md (hunk 1 above)
Comment (SpaceLenore) [reply INSTALL.md]: fixed in commit 7127c28

**Pull request merged**